SCClassLibrary/Common/Core/ExperimentalAbstractObject.sc

```diff
@@ -0,0 +1,74 @@
+// ExperimentalAbstractObject represent the minimum interface needed to get the interpreter to work (it will cause the IDE to post an error as it doesn't support printing).
+// This make it very useful as a wrapper class and to implement some specific features.
+
+// !!! --- PLEASE NEVER ADD A NORMAL PASCAL CASE METHOD TO ABSTRACT OBJECT --- !!!
+// The whole point of this class is to be minimal, ExperimentalAbstractObjects are not Objects, allowing those who subclass it to create their own definition of what an object is and isn't.
+
+ExperimentalAbstractObject {
```

Comment:
Can we call it `AbstractObjectExperimental` ? then it will be found more easily later. It is ok like as it is as well.